modules/virt-pxe-booting-with-mac-address.adoc

```diff
@@ -145,6 +145,7 @@ $ virtctl vnc vmi-pxe-boot
 ----
 $ virtctl console vmi-pxe-boot
 ----
+.Verification
```

Comment:
The `Verification` heading isn't rendering for some reason. Maybe it just needs a line break before it?